website/docs/execution-node/fee-recipient.md

```diff
@@ -291,6 +203,24 @@ See [this issue](https://github.com/prysmaticlabs/prysm/issues/11322) to track t
 
 :::
 
+
+## Frequently asked questions
+
+**How do I know if fee recipient was properly configured?** <br />
+If you don't see any errors after issuing one of the above commands, your fee recipient address has been successfully configured.
+
+**What happened to `fee-recipient-config-file`?** <br />
+`fee-recipient-config-file` and `fee-recipient-config-url` flags are deprecated and have been replaced with `proposer-settings-file` and `proposer-settings-url` flags as of Prysm v2.1.3.
+
+**TODO: what does this mean?** <br />
+- The existing `suggested-fee-recipient` flag must now be used with the new `enable-builder` flag. This allows your validator client to use the MEV Builder API.
```

Comment:
only on the validator side does this need to be done ( just a clarifying point)

Reply:
@james-prysm does this mean that when I set `suggested-fee-recipient` on the validator, I must also set `enable-builder`? Or does it mean "When `enable-builder` is set to `true` on your validator, you must use the `suggested-fee-recipient` to communicate your fee recipient wallet address to builders - JSON/YAML config will not be relayed?"

Reply:
When enable-builder is set to true on your validator, you must use the suggested-fee-recipient to communicate your fee recipient wallet address to builders or use the builder field section in the JSON/YAML